liblouis/liblouis.h.in

```diff
@@ -99,12 +99,14 @@ typedef enum {
 	noContractions = 1,
 	compbrlAtCursor = 2,
 	dotsIO = 4,
-	// for historic reasons 8 is free
-	pass1Only = 16,  // defunct
+	// for historical reasons 8 and 16 are free
```

Comment:
Ugh, sorry but this is ugly as hell. I have a bit of a thing for code aesthetics. Use something along the line of https://stackoverflow.com/a/11041209, i.e. define an array that contains the valid modes and the define a function `isMode` for example that just takes a value, loops through the valid modes and returns true if the given value is contained in the array and false otherwise. 

I can do it if you like.